### DISCUSSION TITLE
Disabled alias/Blocked contact

### DISCUSSION THREAD

**Discussion by trikotret** [simple-login/app#1404] [Q&A] [answered]
Hi
I don't think this feature is working. I did a few tests where I block a sender and selected **Reject** so they can get a notification that the alias doesn't exist. Simple login dashboard shows that email was blocked.

**Comment by 4b15ed24f576** [comment]
Hey, could I ask you to create a support ticket (https://app.simplelogin.io/dashboard/support) so we can better investigate your case?

Thanks a lot

**Reply by trikotret** [reply]
Done!

**Comment by trikotret** [comment, marked as answer]
They got back to me about this bug. I tried it today and now its fixed.

**Reply by trikotret** [reply]
I just tried it today and its not working.

**Reply by JtwoA** [reply]
I just tried this moments ago and it properly hard bounces the message.

_The response from the remote server was:
552 5.2.2 <[alias@subdomain.org](mailto:alias@subdomain.org)>: Recipient address rejected: Address does not exist_

**Reply by trikotret** [reply]
Per support, this was only affecting me. They still working the fix for me
